src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Class WPML_Gutenberg_Strings_In_Block
+ */
+class WPML_Gutenberg_Strings_In_Block {
+
+	/** @var array */
+	private $block_types = array(
```

Comment:
**Note:** These are xpath queries to find the element or attribute that has text needing translation. These will come from `wpml-config.xml` and will be handled in the following merge request.